### PR TITLE
SNES: add missing revisions from no-intro

### PR DESCRIPTION
**Pull request by tjanas**
Add the following missing entries:

```
Atlas, The - Renaissance Voyager (Japan) (Rev 1) (NP).sfc
Battle Robot Retsuden (Japan) (Rev 1).sfc
Dai-3-ji Super Robot Taisen (Japan) (Rev 3).sfc
FIFA 97 (Europe) (En,Fr,De,Es,It,Sv).sfc
Fire Emblem - Thracia 776 (Japan) (NP).sfc
From TV Animation Slam Dunk - Yonkyou Gekitotsu!! (Japan) (Rev 1).sfc
Gojira - Kaijuu Daikessen (Japan) (Rev 1).sfc
Jissen! Pachi-Slot Hisshouhou! 2 (Japan) (Rev 1).sfc
Kamen Rider (Japan) (Rev 1).sfc
Kouryuu Densetsu Villgust - Kieta Shoujo (Japan) (Rev 1).sfc
Magical Quest Starring Mickey Mouse, The (USA) (Rev 1).sfc
Monopoly Game 2, The (Japan) (Rev 1).sfc
Mortal Kombat (USA) (Rev 1).sfc
Numbers Paradise (Japan) (Rev 1).sfc
PowerFest 94 (USA).sfc
Puzzle Bobble - Bust-A-Move (Europe) (Rev 1).sfc
Sangokushi Eiketsuden (Japan) (Rev 1).sfc
Sangokushi III (Japan) (Rev 2).sfc
Shinchou Kouki (Japan) (Rev 1).sfc
Street Fighter II Turbo (Japan) (Rev 1) (Virtual Console).sfc
Super Mario World 2 - Yoshi's Island (Europe) (En,Fr,De) (Rev 2).sfc
Super Street Fighter II (USA) (Rev 1).sfc
Tecmo Super NBA Basketball (USA) (SNS-NA).sfc
Wing Commander (Germany) (Rev 1).sfc
Zero 4 Champ RR (Japan) (Rev 1).sfc
Zootto Mahjong! (Japan) (Rev 1) (NP).sfc
```

@frederic-mahe @SmokeMonsterPacks please review and merge

### CODE REVIEW

File: EverDrive Pack SMDBs/Super EverDrive & SD2SNES SMDB.txt [hunk 1]
```diff
@@ -1449,7 +1451,7 @@ b9d2483ba547b22cb1173ca0895e4fb20b33d1c4cca62526b12d75280dd8501b	SD2SNES/2 Japan
 0c6f0fc84eccbea3473fdff41bcfe100084d6ca8acdbeeabc17bb276bfe039ad	SD2SNES/2 Japan - A-Z/Japan A-C/Cyber Knight (Japan).sfc	1f3ec78d4648ea535027cdefde63429a193d25fd	db9465e77ac9b8a496d0039031c9d446	64034b07
 b1ad7dda4a1540d263bded7180fc786ab4c2e46b953044e0a61883ef223bd2fd	SD2SNES/2 Japan - A-Z/Japan A-C/Cyber Knight II - Chikyuu Teikoku no Yabou (Japan).sfc	2faff9f7082f458d1c9f4f2f1514520f872ff1a1	4244bb90aa722836b746be0574351874	ed8bcbf4
 71e77fdbd9c865ae07e907549b02549ebe87fa8d436401c485e588c61797754a	SD2SNES/2 Japan - A-Z/Japan A-C/Cyborg 009 (Japan).sfc	5ab41e2c6ccce1f40fb9813e8e07a49c785d88c5	0be736bb76d0a5cce320886570382362	4a5263db
-d87a36d2a24f9b2d1b342c6c5985e3054810a04ae06864b4ef52383aac7aa4ef	SD2SNES/2 Japan - A-Z/Japan D-F/Dai-3-ji Super Robot Taisen (Japan) (Rev 2).sfc	5e59151e364d73db030744d2a8b2355bb6fe1e7f	e207d0f97c8d8476bd7b58d81f78864d	c577d666
+2d3ce11c9ba66cba5b5068abea650f1a57883916b822614816c02b6eb8dc2d53	SD2SNES/2 Japan - A-Z/Japan A-C/Dai-3-ji Super Robot Taisen (Japan) (Rev 3).sfc	cc2e8dba67aa3c20450a409630b2ecc8470322bd	1075eb9a226dfbbc8bc8cc2fa3b96d4d	c2941881
```
Comment (frederic-mahe) [inline EverDrive Pack SMDBs/Super EverDrive & SD2SNES SMDB.txt:1454]: Dai is in `Japan A-C`?

File: EverDrive Pack SMDBs/Super EverDrive & SD2SNES SMDB.txt [hunk 2]
```diff
@@ -1154,7 +1155,7 @@ e23b2a8c14fcd1260d35b415ff75c8bd54fea76350517132168e0ceb7666984a	SD2SNES/2 Europ
 ea61ce8e5e977624a8135faa8dbd433f3138fc78b79d473f173bb3e38007de2f	SD2SNES/2 Europe - A-Z/PAL - S-Z/Super Mario All-Stars and Super Mario World (Europe).sfc	be560f0a8de05e432b34be768198ef3defb13cb6	4941387e96ed2841ee3c2de6d1899509	303205d8
 1bdf422695a30e704e8abb7743b9c178d1ef2b200515a83cd41daef85e6b99e2	SD2SNES/2 Europe - A-Z/PAL - S-Z/Super Mario Kart (Europe).sfc	27d9b4f30d39af75075691344b7bdeedbd32ac19	f9fe266e91632e68b558d6b43393eaba	56410e5e
 5cc54b1e5c8d3c7701a5e20514145c3b36f15f26fe0a4fe6d2e43677e4b4eda9	SD2SNES/2 Europe - A-Z/PAL - S-Z/Super Mario World (Europe) (Rev 1).sfc	46bf36be1c3a2ce9de7581323370bd2d891ad5a1	7bb0487eacdb78d6635e89d797e5ab74	b47f5f20
-824f07e93c9ad38fe408af561e8979e3c0211f0c6c98aeb6e6bc85cd6f9edc91	SD2SNES/2 Europe - A-Z/PAL - S-Z/Super Mario World 2 - Yoshi's Island (Europe) (En,Fr,De) (Rev 1).sfc	1e1dc02c684652f9e927f3873af65ea4b374bb80	9f235e4c17cf9762a49b83e82794d278	07e01ef9
+5693f98dceb1fab81cf1aa519607200ca61895d8b3eeb838d11bf655b09834b0	SD2SNES/2 Europe - A-Z/PAL - S-Z/Super Mario World 2 - Yoshi's Island (Europe) (En,Fr,De) (Rev 2).sfc	09402ff319d4517409f961b4012e5eeb1b89ac5c	8de2ee0b0bdbdde16aefedecfb837971	e71b3e72
```
Comment (frederic-mahe) [inline EverDrive Pack SMDBs/Super EverDrive & SD2SNES SMDB.txt:1158]: Just to make sure I understand correctly. Most SNES cartridges have version number `1.0`. Some games have a revision `1.1` (for example `SFRA-K2-1`, the revision of _Secret of Mana_ in France), and a handful have more revisions (up to `1.5` I think). So:
- Title.sfc: means first release,
- Title.sfc (Rev 1): first revision,
- Title.sfc (Rev 2): second revision,
The last revision is the one the pack puts forward. Earlier versions are grouped in a dedicated folder `4 Betas, Prototypes, Revisions`.

Comment (tjanas) [reply EverDrive Pack SMDBs/Super EverDrive & SD2SNES SMDB.txt:1158]: I believe your understanding is correct. There may be revisions where it isn't obvious which is the newest release:
[Tecmo Super NBA Basketball (USA) (SNS-NA)](https://datomatic.no-intro.org/?page=show_record&s=49&n=3734)
[Tecmo Super NBA Basketball (USA) (SNS-XM)](https://datomatic.no-intro.org/?page=show_record&s=49&n=3003)

File: EverDrive Pack SMDBs/Super EverDrive & SD2SNES SMDB.txt (hunk 1 above)
Comment (tjanas) [reply EverDrive Pack SMDBs/Super EverDrive & SD2SNES SMDB.txt:1454]: Nice catch, I will fix it.

**Pull request merged**